### ISSUE TITLE
Weather API: Yahoo GEO place finder YQL not supported

### ISSUE THREAD

**Issue by DisplayMonkey** [fuel9/DisplayMonkey#9]
This call:

`http://query.yahooapis.com/v1/public/yql?q=select+*+from+geo.placefinder+where+text%3D%2233.52068%2C-86.81176%22+and+gflags%3D%22R%22`

Results in the following XML:

```
<?xml version="1.0" encoding="UTF-8"?>
-<query yahoo:lang="en-US" yahoo:created="2017-02-03T18:46:49Z" yahoo:count="0" xmlns:yahoo="http://www.yahooapis.com/v1/base.rng">
<!-- total: 2 -->
<!-- prod_bf1_1;paas.yql;queryyahooapiscomproductionbf1;8055ca03-e7ed-11e6-bcc3-d4ae52974741 -->
<meta>
<message>geo.placefinder table is no longer supported</message>
</meta>
<results/>
</query>
```

**Comment by DisplayMonkey** [comment]
More info here:
https://forums.developer.yahoo.net/discussion/7844/yql-geo-placefinder-is-now-shutdown

**Comment by DisplayMonkey** [comment]
fixed in 1.3.3

**Comment by jaschrock** [comment]
I have around 20 locations that are sharing the same canvas. We are starting to deploy these in production. Things are fine up to around my 12th conference room. For some reason, conference room 12 through 15 all show as conference room 11. I checked the locations, they are all set up uniquely. There is one frame, an outlook calendar that varies between the location. This is the same for all of them. The title also is the same so I don't think it has to do with the calendar. I see no area that is set up incorrectly. 

I have double checked the designated location on the calendar. I have made sure the location specified points to the correct area. I can't think of anywhere else to look. Thoughts? 

Thanks

Alan
